### ISSUE TITLE
PassBolt Migration failed

### ISSUE THREAD

**Issue by GlisseManTV** [truenas/apps#610]
I tried to migrate a test environment from fresh scale 24.04 with latest passbolt installation (from truenas catalog) to 24.10.
I reach this error and I can't resolve it quickly.

```
2024-10-03 20:44:01.951057+00:002024-10-03 20:44:01+00:00 [ERROR] [Entrypoint]: Unable to start server.
2024-10-03 20:44:02.299536+00:002024-10-03 20:44:02+00:00 [Note] [Entrypoint]: Entrypoint script for MariaDB Server 1:10.11.9+maria~ubu2204 started.
2024-10-03 20:44:02.469100+00:002024-10-03 20:44:02+00:00 [Note] [Entrypoint]: Starting temporary server
2024-10-03 20:44:02.469968+00:002024-10-03 20:44:02+00:00 [Note] [Entrypoint]: Waiting for server startup
2024-10-03 20:44:02.503216+00:002024-10-03 20:44:02 0 [Note] Starting MariaDB 10.11.9-MariaDB-ubu2204 source revision 0e8fb977b00983d98c4c35e39bc1f36463095938 server_uid GLV6Z50+HNgCl0g1vN+VLou9/SE= as process 27
2024-10-03 20:44:02.512321+00:002024-10-03 20:44:02 0 [Note] InnoDB: Compressed tables use zlib 1.2.11
2024-10-03 20:44:02.513553+00:002024-10-03 20:44:02 0 [Note] InnoDB: Number of transaction pools: 1
2024-10-03 20:44:02.513582+00:002024-10-03 20:44:02 0 [Note] InnoDB: Using crc32 + pclmulqdq instructions
2024-10-03 20:44:02.513677+00:002024-10-03 20:44:02 0 [Warning] mariadbd: io_uring_queue_init() failed with errno 1
2024-10-03 20:44:02.513770+00:002024-10-03 20:44:02 0 [Warning] InnoDB: liburing disabled: falling back to innodb_use_native_aio=OFF
2024-10-03 20:44:02.514343+00:002024-10-03 20:44:02 0 [Note] InnoDB: Initializing buffer pool, total size = 128.000MiB, chunk size = 2.000MiB
2024-10-03 20:44:02.528258+00:002024-10-03 20:44:02 0 [Note] InnoDB: Completed initialization of buffer pool
2024-10-03 20:44:02.529270+00:002024-10-03 20:44:02 0 [Note] InnoDB: Buffered log writes (block size=512 bytes)
2024-10-03 20:44:02.529535+00:002024-10-03 20:44:02 0 [ERROR] InnoDB: Upgrade after a crash is not supported. The redo log was created with MariaDB 10.6.14. You must start up and shut down MariaDB 10.7 or earlier.
2024-10-03 20:44:02.529562+00:002024-10-03 20:44:02 0 [ERROR] InnoDB: Plugin initialization aborted with error Generic error
2024-10-03 20:44:02.529568+00:002024-10-03 20:44:02 0 [Note] InnoDB: Starting shutdown...
2024-10-03 20:44:02.731758+00:002024-10-03 20:44:02 0 [ERROR] Plugin 'InnoDB' registration as a STORAGE ENGINE failed.
2024-10-03 20:44:02.731799+00:002024-10-03 20:44:02 0 [Note] Plugin 'FEEDBACK' is disabled.
2024-10-03 20:44:02.731836+00:002024-10-03 20:44:02 0 [ERROR] Unknown/unsupported storage engine: InnoDB
2024-10-03 20:44:02.731874+00:002024-10-03 20:44:02 0 [ERROR] Aborting
```

I tried to stop mariadb container and run another with the same credentials with the 10.6 version requested but it make no change.

**Comment by stavros-k** [comment]
Hello, please follow this steps:

https://github.com/truenas/apps/issues/592#issuecomment-2391455422

Let me know if that worked.

**Comment by GlisseManTV** [comment]
Hi ! 

Thanks for the update !

Yes, it work now.
I got the path with `sudo docker inspect ix-passbolt-mariadb-1`

Little warning that CTRL+C doesn't stop the temp-container.
I must stop it manually with `sudo docker stop passbolt-tempdb`

I tried it 3 times, and with/out data in passbolt.
Sometimes I must rechoose the certificate in the UI to make passbolt container working.
No losing data.

Great thanks !
